samples/word/50-document/manage-shapes-text-boxes.yaml

```diff
@@ -0,0 +1,276 @@
+
+        // Default helper for invoking an action and handling errors.
+        async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-Fabric ms-font-m">
+            This sample demonstrates how to work with shapes and text boxes.
+        </section>
+        <section class="ms-Fabric setup ms-font-m">
+            <h3>Set up</h3>
+            <button id="setup" class="ms-Button">
+              <span class="ms-Button-label">Add text</span>
+          </button>
+        </section>
+        <section class="ms-Fabric samples ms-font-m">
+            <h3>Try it out</h3>
+            <button id="insert-text-box-into-current-selection" class="ms-Button">
+                <span class="ms-Button-label">Insert text box into current selection</span>
+            </button>
+            <button id="get-text-boxes-in-main-doc" class="ms-Button">
+              <span class="ms-Button-label">Get text boxes in main document</span>
+          </button>
+            <button id="insert-text-box-in-header-at-first-paragraph" class="ms-Button">
+              <span class="ms-Button-label">Insert text box into header</span>
+          </button>
+            <button id="get-text-box-in-header" class="ms-Button">
+              <span class="ms-Button-label">Get text boxes in header</span>
+          </button>
```

Comment:
Since there are so many buttons, consider putting the buttons for the header text box in a different section.